library/src/main/java/com/airbnb/android/airmapview/RuntimePermissionUtils.java

```diff
@@ -36,6 +40,27 @@ private static boolean verifyPermissions(int... grantResults) {
   }
 
   /**
+   * Check the permissions are the requested permissions
+   *
+   * @param permissions
+   * @return true if the permissions are matched perfectly
+   */
+  static boolean isRequestedPermission(String[] permissions) {
+    if (permissions.length != LOCATION_PERMISSIONS.length) {
+      return false;
+    }
+
+    final Set<String> requestPermissions = new HashSet<>(Arrays.asList(LOCATION_PERMISSIONS));
```

Comment:
it's not clear to me why you're creating this set with a single item inside?
can you just check if `permission.equals(LOCATION_PERMISSIONS)` on line `55`?